### PR TITLE
unit testing code for vrfs module added

### PR DESCRIPTION
**Pull request by kannankvs**
##### SUMMARY
unit testing code for vrfs module added

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
vrfs module unit testing

### CODE REVIEW

File: tests/unit/modules/network/sonic/fixtures/sonic_vrfs.yaml
Comment (kerry-meyer) [inline tests/unit/modules/network/sonic/fixtures/sonic_vrfs.yaml:1]: Please fix the whitespace errors in this file. They are causing Ansible sanity test failures.

**Review by kerry-meyer — CHANGES_REQUESTED** [review]
Please fix the whitespace errors in the tests/unit/modules/network/sonic/fixtures/sonic_vrfs.yaml file. They are causing Ansible sanity test failures.

**Comment by kannankvs** [comment]
An issue [194] (https://github.com/ansible-collections/dellemc.enterprise_sonic/issues/194)  is observed in VRF module. UT will succeed only when the issue is resolved. Need to wait until the issue is resolved.

File: tests/unit/modules/network/sonic/fixtures/sonic_vrfs.yaml
Comment (kannankvs) [reply tests/unit/modules/network/sonic/fixtures/sonic_vrfs.yaml:1]: Addressed as suggested.

**Review by kerry-meyer — APPROVED** [review]

**Pull request merged**